tools/cli/workflow_commands.go

```diff
@@ -53,7 +53,8 @@ import (
 func RestartWorkflow(c *cli.Context) error {
 	wfClient, err := getWorkflowClient(c)
 	if err != nil {
-		return err
+		fmt.Println("Unable to Get Workflow Client: ", err)
```

Comment:
It's probably better to use commoncli.Problem so that errorHelp() is called automatically
```suggestion
  return commoncli.Problem("Unable to Get Workflow Client: ", err)
```